Modules/Scripted/VPAWModel/Resources/UI/VPAWModel.ui

```diff
@@ -38,181 +38,94 @@
-      <string>VPAW Model: Inputs</string>
+    <widget class="ctkCollapsibleButton" name="installPediatricAirwayAtlasCollapsibleButton" native="true">
+     <property name="text" stdset="0">
+      <string>VPAW Model: Install Pediatric Airway Atlas</string>
      </property>
-     <layout class="QFormLayout" name="formLayout_2">
+     <property name="collapsed" stdset="0">
+      <bool>false</bool>
+     </property>
+     <layout class="QFormLayout" name="installPediatricAirwayAtlasFormLayout">
       <item row="0" column="0">
-       <widget class="QLabel" name="label">
+       <widget class="QLabel" name="pediatricAirwayAtlasDirectoryLabel">
         <property name="text">
-         <string>Input volume:</string>
+         <string>Pediatric Airway Atlas source directory</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
-       <widget class="qMRMLNodeComboBox" name="inputSelector">
-        <property name="toolTip">
-         <string>Pick the input to the algorithm.</string>
-        </property>
-        <property name="nodeTypes">
-         <stringlist notr="true">
-          <string>vtkMRMLScalarVolumeNode</string>
-         </stringlist>
-        </property>
-        <property name="showChildNodeTypes">
-         <bool>false</bool>
-        </property>
-        <property name="addEnabled">
-         <bool>false</bool>
-        </property>
-        <property name="removeEnabled">
-         <bool>false</bool>
+       <widget class="ctkPathLineEdit" name="PediatricAirwayAtlasDirectory">
+        <property name="filters">
+         <set>ctkPathLineEdit::Dirs</set>
         </property>
        </widget>
       </item>
-      <item row="2" column="0">
-       <widget class="QLabel" name="label_3">
-        <property name="text">
-         <string>Image threshold:</string>
+      <item row="1" column="0" colspan="2">
+       <widget class="QPushButton" name="installPediatricAirwayAtlasButton">
+        <property name="enabled">
+         <bool>false</bool>
         </property>
-       </widget>
-      </item>
-      <item row="2" column="1">
-       <widget class="ctkSliderWidget" name="imageThresholdSliderWidget">
         <property name="toolTip">
-         <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
-        </property>
-        <property name="singleStep">
-         <double>0.100000000000000</double>
-        </property>
-        <property name="minimum">
-         <double>-100.000000000000000</double>
-        </property>
-        <property name="maximum">
-         <double>500.000000000000000</double>
+         <string>Install Pediatric Airway Atlas and Dependencies</string>
         </property>
-        <property name="value">
-         <double>0.500000000000000</double>
+        <property name="text">
+         <string>Install Pediatric Airway Atlas and Dependencies</string>
```

Comment:
The name of this button is a bit misleading. It does two things:

- installs dependencies into the python environment
- adds paa to the `sys.path`

The first one is "installing" and we only need to do it once.

The second one is not really "installing" and we need to do it every time.

The name "Install" of this button makes it sound like we only need to do it once.

We should come up with a clearer name. One suggestion: "Link to Pediatric Airway Atlas Codebase"

Reply:
Will do.